Runar/Runar/SwiftGen/Fonts.swift

```diff
@@ -1,7 +1,7 @@
 // swiftlint:disable all
 // Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
-#if os(OSX)
+#if os(macOS)
```

Comment:
What is? Why do you change it?

Reply:
After pod update it's changed I supposed.